ClientApp/viking-vault/src/UserDashboard/AutomaticPaymentList.tsx

```diff
@@ -1,10 +1,16 @@
 import  React  from 'react';
 import {constants} from "../Resources/Constants";
 import './ViewAutomaticPayments.css'
+import {CreateAutomaticPaymentForm} from "./CreateAutomaticPaymentModal"
 
 const baseUrl = constants.baseUrl + "AutomaticPayment";
 
+interface IModals {
+    openCreateAutomaticPaymentModal: boolean
+}
+
 interface IAutomaticPayment {
+    companyId: number,
     companyName: string,
     amount: Number,
```

Comment:
Please use "number" for amount instead of "Number" and eventually investigate the difference between these two.